aspnet/mvc/views/forms/sample/final/Views/DemoReg/Register.cshtml

```diff
@@ -0,0 +1,20 @@
+﻿@model FormsTH.ViewModels.RegisterViewModel
+@addTagHelper "*, Microsoft.AspNet.Mvc.TagHelpers"
+
+<form asp-controller="DemoReg" asp-action="Register" method="post">
+    <div asp-validation-summary="ValidationSummary.All"></div>
```

Comment:
@dougbu  I  copied this from the views in the account folder (generated by our templates). They use both property level  ``asp-validation-for="Email"`` and  ``  <div asp-validation-summary="ValidationSummary.All" class="text-danger"></div>``
I've removed the summary from my sample. Should I open an issue on the templates?